File: payroll-api/src/main/java/com/company/payroll/model/StaffLeaveDetails.java
package com.company.payroll.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@RequiredArgsConstructor
public class StaffLeaveDetails {
    private Integer ldId;
    private Integer annualLeave;
    private Integer hospitalityLeave;
    private Integer maternityLeave;
    private Integer unpaidLeave;
}

File: kotlin-payroll/src/main/kotlin/com/company/payroll/model/StaffLeaveDetails.kt
package com.company.payroll.model

import com.company.payroll.annotation.NoArg
import lombok.NoArgsConstructor

/**
 * @author caroline
 *
 * <p> Created: 22 Jun 2023
 * <p> Use as model for table staff_leave_details
 */
@NoArg
data class StaffLeaveDetails(
    var ldId: Int,
    var annualLeave: Int,
    var hospitalityLeave: Int,
    var maternityLeave: Int,
    var unpaidLeave: Int
)
